docker/openemr/flex-3.11/utilities/devtools.sh

```diff
@@ -0,0 +1,25 @@
+#! /bin/sh
+
+
+if [ $1 == "psr2-report" ]; then
```

Comment:
also, you could create a "clean-sweep" call that will run all the tests. The all the ifs, would be if blah or "clean-sweep".

Reply:
so would also drop all the elif stuff and just use if for all the conditionals

Reply:
Seen, let me revert to this